src/auspex/qubit/qubit_exp.py

```diff
@@ -415,6 +418,9 @@ def init_instruments(self):
         for gen_proxy in self.generators:
             gen_proxy.instr.output = True
 
+        for dc_source_proxy in self.dc_sources:
+            dc_source_proxy.instr.output = True
```

Comment:
This should not be necessary if the `DCSource` `output` field is `True` (you can set when creating the instrument or later) see [here](https://github.com/BBN-Q/bbndb/blob/44014d47684b4cf7260a7ac65f7430ec368cc9d3/bbndb/qgl.py#L145)